opennav_coverage/include/opennav_coverage/visualizer.hpp

```diff
@@ -46,16 +46,16 @@ class Visualizer
   {
     nav_plan_pub_ = rclcpp::create_publisher<nav_msgs::msg::Path>(
       node->get_node_topics_interface(),
-      "coverage_server/coverage_plan", rclcpp::QoS(1));
+      "coverage_server/coverage_plan", rclcpp::QoS(1).transient_local());
```

Comment:
Update rviz config to match

Reply:
I don't know about this one.  There is no rviz config it this repository that I can see.  It was a change by @tonynajjar

Reply:
Huh. I could have sworn someone added that recently 

Edit: oh, it was only for humble :roll_eyes:  https://github.com/open-navigation/opennav_coverage/commit/73fc308920bb4a71c23af39cb12086a528da047a

Reply:
I cherry-picked the commit and amended the coverage_server/coverage_plan Durability Policy to Transient Local

https://github.com/open-navigation/opennav_coverage/pull/53/commits/9a38dc4ab8c7ad2a851f8ff04b2fa0c13974bab2